wp-content/plugins/zippicks-social/includes/class-follow-manager.php

```
@@ -0,0 +1,645 @@
+            [
+                'actor_id' => $actor_id,
+                'actor_type' => 'user',
+                'action' => $action,
+                'object_type' => $object_type,
+                'object_id' => $object_id,
+                'visibility' => 'public',
+                'created_at' => current_time('mysql')
+            ],
+            ['%d', '%s', '%s', '%s', '%d', '%s', '%s']
+        );
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'log_activity'. (Unused Code Rules)

(UnusedPrivateMethod)